pages.pt_BR/common/cupsdisable.md

```diff
@@ -0,0 +1,13 @@
+# cupsdisable
+
+> Stop printers and classes.
+> See also: `cupsenable`, `cupsaccept`, `cupsreject`, `lpstat`.
+> More information: <https://www.cups.org/doc/man-cupsenable.html>.
+
+- Stop one or more destination(s):
```

Comment:
What is a destination? A printer?

Reply:
> What is a destination? A printer?

@gutjuri a printer or a class of printers.

Reply:
In my opinion, this should be clarified in the page.

Reply:
@gutjuri I think we could use `{{printer|class}}`, right?

Reply:
As "destination" is used all over the place, could we put a NOTE in the beginning of the command description?
Like: `NOTE: destination is referred as a printer or a class of printers`

What do you guys think @gutjuri @vitorhcl ?

Reply:
I think `{{printer|class}}` syntax would pollute the descriptions of a bunch of commands.